### PR TITLE
fix: 🐛 ignore frames without SourceImageSequence information when loading a segmentation

### PR DESCRIPTION
**Pull request by Punzo**
IDC issue: https://github.com/OHIF/Viewers/issues/2345

Two main limitations are generated by the current cornerstone approach (segmentation geometry has to fit the one of the source images):

1) we can't have out of plane frames;
2) we can't have segmentation frames with different Columns and Rows attributes respect to the source frames.

in this PR we add checks to detect such data and abort the laoding.

Relevant IDC datasets:

- http://localhost:3000/projects/idc-dev-etl/locations/us-central1/datasets/idc/dicomStores/v2/study/1.3.6.1.4.1.14519.5.2.1.7311.5101.758555921127856064761732086690?seriesInstanceUID=1.3.6.1.4.1.14519.5.2.1.7311.5101.165636727522505811947315188478

### CODE REVIEW

**Comment by Punzo** [comment]
@pieper @igoroctaviano can you review and merge this please?

**Review by pieper — APPROVED** [review]
LGTM 👍

File: src/adapters/Cornerstone/Segmentation_4X.js [hunk 1]
```diff
@@ -1059,6 +1072,21 @@ function insertPixelDataPlanar(
             continue;
         }
 
+        const sourceImageMetadata = cornerstone.metaData.get(
```
Comment (rogerbramon) [inline src/adapters/Cornerstone/Segmentation_4X.js:1075]: Using `cornerstone.metaData` can cause errors if `cornerstone` is not globally available (when using webpack, for instance). Should `metadataProvider` option used instead? There are 3 occurrences of `cornerstone.metaData` in this file on master branch. @pieper

Comment (Punzo) [reply src/adapters/Cornerstone/Segmentation_4X.js:1075]: @rogerbramon nice catch, fix applied in https://github.com/dcmjs-org/dcmjs/pull/280. @pieper please have a look

**Pull request merged**